package.json

```diff
@@ -17,7 +17,7 @@
 		"lint:styles": "stylelint \"**/*.css\"",
 		"prepare": "husky install",
 		"test": "playwright test --headed",
-		"test:ci": "playwright test",
+		"test:ci": "playwright test --project=chromium",
```

Comment:
Only run the chromium project, https://github.com/philly-js-club/philly-js-club-website-remix/issues/8#issuecomment-1526176335. Thanks @RNR1! I also only modified the `ci` job in case you still want local to run all jobs.